### PR TITLE
PP-7011 Adding MOTO masking permissions

### PR DESCRIPTION
**Pull request by narinderchana**
Description:
- Adding permissions for reading and updating MOTO mask input settings

### CODE REVIEW

File: src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:moto_masking_account_update_permission
```
Comment (alexbishop1) [inline src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql:3]: Changeset ID doesn’t quite match the permission: should be `uk.gov.pay:moto_mask_input_update_permission`.

File: src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+INSERT INTO permissions(id, name, description) VALUES (49, 'moto-mask-input:update', 'Update Moto Masking Account');
```
Comment (alexbishop1) [inline src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql:4]: The description should be something like “Update MOTO mask input settings”.

File: src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+INSERT INTO  role_permission VALUES ((SELECT id FROM roles WHERE name = 'super-admin'), (SELECT id FROM permissions WHERE name = 'moto-mask-input:update'), NOW(), NOW());
+INSERT INTO  role_permission VALUES ((SELECT id FROM roles WHERE name = 'admin'), (SELECT id FROM permissions WHERE name = 'moto-mask-input:update'), NOW(), NOW());
+
+--changeset uk.gov.pay:moto_masking_account_read_permission
```
Comment (alexbishop1) [inline src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql:8]: Changeset ID doesn’t quite match the permission: should be `uk.gov.pay:moto_mask_input_read_permission`.

File: src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+INSERT INTO  permissions(id, name, description) VALUES (50, 'moto-mask-input:read', 'View Moto Masking Account');
```
Comment (alexbishop1) [inline src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql:9]: The description should be something like “View MOTO mask input settings”.

**Review by alexbishop1 — CHANGES_REQUESTED** [review]
Looks fine apart from a few description strings.

File: src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql (hunk 1 above)
Comment (narinderchana) [reply src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql:3]: ok will make that change

File: src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql (hunk 2 above)
Comment (narinderchana) [reply src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql:4]: ok will make that change

File: src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql (hunk 3 above)
Comment (narinderchana) [reply src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql:8]: ok will make that change

File: src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql (hunk 4 above)
Comment (narinderchana) [reply src/main/resources/migrations/00063_add_moto_account_masking_permisions.sql:9]: ok will make that change

**Review by alexbishop1 — APPROVED** [review]

**Pull request merged**